ansible/vars/icds/icds_public.yml

```diff
@@ -175,7 +175,7 @@ etc_host_lines:
   - '10.247.24.13		www.icds-cas.gov.in'
 etc_host_lines_removed: ['etc_host_lines']
 
-http_proxy_address: '10.247.63.132'
+http_proxy_address: '10.247.24.16'
```

Comment:
According to cal This changed because the new machines have not been setup with the patch server so this tunnels it through es0 to the proxy. I needed this change for celery1, but i'm not sure that this should get commited, if it's not hard to setup the patch server to work with new machines